src/common/interop/HotkeyManager.cpp

```diff
@@ -1,94 +1,89 @@
+        return true;
     }
-}
+    bool HotkeyManager::FilterKeyboardProc(KeyboardEvent ev)
+    {
+        // Updating the pressed keys here so we know if the keypress event should be propagated or not.
+        pressedKeys.Win = (GetAsyncKeyState(VK_LWIN) & 0x8000) || (GetAsyncKeyState(VK_RWIN) & 0x8000);
+        pressedKeys.Ctrl = GetAsyncKeyState(VK_CONTROL) & 0x8000;
+        pressedKeys.Alt = GetAsyncKeyState(VK_MENU) & 0x8000;
+        pressedKeys.Shift = GetAsyncKeyState(VK_SHIFT) & 0x8000;
+        pressedKeys.Key = static_cast<unsigned char>(ev.key);
 
-// Hotkeys are intended to be global, therefore they are always active no matter the
-// context in which the keypress occurs.
-bool HotkeyManager::IsActiveProc()
-{
-    return true;
-}
+        // Convert to hotkey handle
+        auto pressedKeysHandle = GetHotkeyHandle(pressedKeys);
 
-// KeyboardEvent callback is only fired for relevant key events.
-bool HotkeyManager::FilterKeyboardProc(KeyboardEvent ^ ev)
-{
-    // Updating the pressed keys here so we know if the keypress event should be propagated or not.
-    pressedKeys->Win = (GetAsyncKeyState(VK_LWIN) & 0x8000) || (GetAsyncKeyState(VK_RWIN) & 0x8000);
-    pressedKeys->Ctrl = GetAsyncKeyState(VK_CONTROL) & 0x8000;
-    pressedKeys->Alt = GetAsyncKeyState(VK_MENU) & 0x8000;
-    pressedKeys->Shift = GetAsyncKeyState(VK_SHIFT) & 0x8000;
-    pressedKeys->Key = static_cast<unsigned char>(ev->key);
+        // Check if any hotkey matches the pressed keys if the current key event is a key down event
+        if ((ev.message == WM_KEYDOWN || ev.message == WM_SYSKEYDOWN) && hotkeys.find(pressedKeysHandle)!=hotkeys.end())
+        {
+            return true;
+        }
 
-    // Convert to hotkey handle
-    auto pressedKeysHandle = GetHotkeyHandle(pressedKeys);
+        return false;
+    }
 
-    // Check if any hotkey matches the pressed keys if the current key event is a key down event
-    if ((ev->message == WM_KEYDOWN || ev->message == WM_SYSKEYDOWN) && hotkeys->ContainsKey(pressedKeysHandle))
+    uint16_t HotkeyManager::RegisterHotkey(winrt::PowerToys::Interop::Hotkey const& _hotkey, winrt::PowerToys::Interop::HotkeyCallback const& _callback)
     {
-        return true;
+        auto handle = GetHotkeyHandle(_hotkey);
+        hotkeys[handle] = _callback;
+        return handle;
     }
 
-    return false;
-}
-
-// NOTE: Replaces old hotkey if one already present.
-HOTKEY_HANDLE HotkeyManager::RegisterHotkey(Hotkey ^ hotkey, HotkeyCallback ^ callback)
-{
-    auto handle = GetHotkeyHandle(hotkey);
-    hotkeys[handle] = callback;
-    return handle;
-}
+    void HotkeyManager::UnregisterHotkey(uint16_t _handle)
+    {
+        auto iter = hotkeys.find(_handle);
+        if (iter != hotkeys.end()) {
+            hotkeys.erase(iter);
+        }
+    }
 
-void HotkeyManager::UnregisterHotkey(HOTKEY_HANDLE handle)
-{
-    hotkeys->Remove(handle);
-}
+    void HotkeyManager::Close()
```

Comment:
Since HotkeyManager before also seemed to be "disposeable", thought to include the "ICloseable" interface, which means it needs to define the Close. (That will be the Dispose in C#).